ger_deps

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -e
+
+cd
+sudo rm -rf /tmp/installitrude/
+mkdir /tmp/installitrude
+cd /tmp/installitrude
+clear
+
+# INSTALLATION OF LUAROCKS
+wget https://luarocks.org/releases/luarocks-3.9.2.tar.gz
+tar zxpf luarocks-3.9.2.tar.gz
+cd luarocks-3.9.2
+./configure && make && sudo make install
+echo -e "\n\e[32mLuarocks Installed successfully\e[0m\n"
+
+# INSTALLATION OF LUAFILESYSTEM
+sudo luarocks install luafilesystem
+echo "require \"lfs\"" | lua
+echo -e "\n\e[32mLuaFileSystem Installed successfully\e[0m\n"
+sleep 0.5
+clear
```

Comment:
I don't like that clear with the sleep at all, We really shouldn't do that in my opinion. Just let the user see what the script they ran just did.

Reply:
it's there for a reason. You see, after that I launch the main menu again which is formatted to be on a clear terminal.

But if you prefer we can say "Press Enter to continue"